content/docs/error-boundaries.md

```diff
@@ -116,21 +117,21 @@ try {
 }
 ```
 
-However, React components are declarative and specify *what* should be rendered:
+Ancak, React bileşenleri bildirimseldir ve **neyin** render edileceğini belirlerler:
 
 ```js
 <Button />
 ```
 
-Error boundaries preserve the declarative nature of React, and behave as you would expect. For example, even if an error occurs in a `componentDidUpdate` method caused by a `setState` somewhere deep in the tree, it will still correctly propagate to the closest error boundary.
+Hata sınırları React'in bildirimsel doğasını korur ve beklendiği gibi davranır. Örneğin, eğer ağacın derinlerinde bir yerde `componentDidUpdate` içinde `setState`'in yol açtığı bir hata oluşsa bile en yakın hata sınırına doğru bir şekilde yönlendirilir.
```

Comment:
We can say `bileşen ağacı` instead of just `agac`. it is better to be more explicit:

> Örneğin, eğer bileşen ağacının derinlerinde ....
 
Remember: There are several places includes `agac` on this page. Consider updating all.